### DISCUSSION TITLE
How do I reference a "Counter", so that I can use them in scripts and other modules

### DISCUSSION THREAD

**Discussion by JoueurSansFromage** [Larkinabout/fvtt-custom-dnd5e#130] [Q&A]
Like the title said! 

I'd like to track my player's reputation from the counters, and use that counter in other scripts, macros and modules. How do I reference it? I tried simply taking the UUID or the @"counter" and it doesnt seem to work!

**Comment by Larkinabout** [comment]
They're stored as flags against the actor. So, if your counter key is `reputation`, you could do something like: `game.actors.getName("Actor Name").getFlag("custom-dnd5e", "reputation");`.

**Comment by JoueurSansFromage** [comment]
I'm trying to combo with TheRipper's Skill tree, and I need to reference as an "Active Effet" for the "skill requirement" to work (I want my players to have a certain treshold of reputation to be able to purchase certain skills. I don't know how I would manage to do that. I've tried your proposition:

game.actors.getName("Ludivine").getFlag("custom-dnd5e", "Reputation") but it doesnt seem to work.

is there a way to reference the selected token like AC with "system.attributes.ac.value", which simply checks the token's AC value, I could have something like that for the "Reputation" counter?

If that is impossible I'll simply check for an other way to track it but Counters seemed a simple way for my players to use

**Comment by Larkinabout** [comment]
Flags aren't generally available in that way, unless you have a module that adds them to the roll data. For example, the Dynamic Active Effects module makes flag data available for Active Effects.

**Reply by JoueurSansFromage** [reply]
I'll explore that direction then with DAE! Thanks
